app/src/main/res/layout/player.xml

```diff
@@ -305,6 +305,21 @@
                         app:srcCompat="@drawable/ic_language"
                         tools:ignore="RtlHardcoded" />
 
+                    <androidx.appcompat.widget.AppCompatImageButton
+                        android:id="@+id/getUrlButton"
+                        android:layout_width="wrap_content"
+                        android:layout_height="35dp"
+                        android:layout_marginEnd="8dp"
+                        android:background="?attr/selectableItemBackground"
+                        android:clickable="true"
+                        android:contentDescription="@string/get_url"
+                        android:focusable="true"
+                        android:padding="@dimen/player_main_buttons_padding"
+                        android:scaleType="fitXY"
+                        app:tint="@color/white"
+                        app:srcCompat="@drawable/ic_edit"
```

Comment:
Why ic_edit?

Reply:
Just because I needed an icon, and edit seems accurate enough, as a place holder at least. It also looks like copy to edit or something.

Reply:
Add the `content_copy` icon: https://fonts.google.com/icons?selected=Material%20Icons%3Acontent_copy
You can add it seamlessly in Android Studio by right-clicking on `res` -> New -> Vector asset
Then follow the dialog. Add two icons: a white one to the `drawable` folder and a black one to the `drawable-night` folder. Name them `ic_content_copy.xml`.